### PR TITLE
Wizard recipe: PSRDADA-v1.0.0

### PR DESCRIPTION
**Pull request by kiranshila**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: PSRDADA
* Version: v1.0.0

@staticfloat please review and merge.

### CODE REVIEW

**Comment by kiranshila** [comment]
This is for the linux-only ringbuffer IPC library [PSRDADA](http://psrdada.sourceforge.net/). Unfortunately, it uses quite a few low-level intrinsics to x86 that prevent it from working on ARM. I'm actually pretty surprised it built on i686.

**Comment by kiranshila** [comment]
Anyone get a change to look at this?

**Comment by kiranshila** [comment]
@giordano Is this PR good to go?

File: P/PSRDADA/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://sourceforge.net/code-snapshots/git/p/ps/psrdada/code.git/psrdada-code-62fc33a8730ffa2611d955e732ce7250a5a055b5.zip", "dfe8e8a9b66766a4c84836de26150dfc9bc204bc596b0c5788905e26dda2f1db")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd psrdada-code-62fc33a8730ffa2611d955e732ce7250a5a055b5/
```
Comment (giordano) [inline P/PSRDADA/build_tarballs.jl:16]:
```suggestion
cd $WORKSPACE/srcdir/psrdada*
```

File: P/PSRDADA/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release~
```
Comment (giordano) [inline P/PSRDADA/build_tarballs.jl:17]:
```suggestion
cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release
```

File: P/PSRDADA/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    ExecutableProduct("dada_dbnic", :dada_dbnic),
+    ExecutableProduct("dada_edit", :dada_edit),
+    ExecutableProduct("dada_junkdb", :dada_junkdb),
+    ExecutableProduct("dada_pwc_demo", :dada_pwc_demo),
+    ExecutableProduct("dada_write_block_test", :dada_write_block_test)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="GSL_jll", uuid="1b77fbbe-d8ee-58f0-85f9-836ddc23a7a4"))
```
Comment (giordano) [inline P/PSRDADA/build_tarballs.jl:70]:
```suggestion
    Dependency(PackageSpec(name="GSL_jll", uuid="1b77fbbe-d8ee-58f0-85f9-836ddc23a7a4"); compat="~2.7.2")
```

**Pull request merged**